src/test/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraphTest.kt

```diff
@@ -45,8 +44,12 @@ class HaplotypeGraphTest {
             }
         }
 
-        Sizeof.printMemoryUse()
+    }
 
+    @Test
```

Comment:
This now needs a test with a diploid hvcf.

Reply:
Do we have input for that? I thought we decided to get the haploid working first. Then make sure diploid works after push to release in December?

Reply:
It will be easy enough to modify an existing test hvcf. The test hvcf should include missing allele calls (e.g., A/., ./.). But, it can wait to be added later and does not have to part of this PR.